src/openzaak/components/zaken/admin/zaken.py

```diff
@@ -111,6 +120,18 @@ class StatusAdmin(AuditTrailAdminMixin, UUIDAdminMixin, admin.ModelAdmin):
     viewset = "openzaak.components.zaken.api.viewsets.StatusViewSet"
     inlines = [SubStatusForStatusInline]
 
+    def save_model(self, request, obj, form, change):
+        dataref = obj.get_absolute_api_url()
```

Comment:
The dataref should be the relative URL of the Zaak
```suggestion
        dataref = obj.zaak.get_absolute_api_url()
```